lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart

```diff
@@ -85,7 +85,7 @@ class ExpansionContactListTile extends StatelessWidget {
                             ),
                           ],
                           const SizedBox(width: 8.0),
-                          if (contact.status != null)
+                          if (contact.status == ContactStatus.inactive)
```

Comment:
What happens when contact status is null

Reply:
Added null check